docs/howto/use_pum_for_migrations.rst

```diff
@@ -0,0 +1,101 @@
+Use Pum for schema migrations
+-----------------------------
+
+When Procrastinate developers make changes to the Procrastinate database schema they
+write migration scripts.
+
+Here's an example of a migration script:
+
+.. code-block:: sql
+
+    ALTER TABLE procrastinate_jobs ADD COLUMN extra TEXT;
+
+Migration scripts are pure-SQL scripts, meaning that they may be applied to the database
+using any PostgreSQL client, including ``psql`` and ``PGAdmin``.
+
+The names of migration script files adhere to a certain pattern, which comes from the
+`Pum`_ project. Pum is thus a good choice for Procrastinate migrations, although other
+migration tools may be used as well.
+
+See the `Pum README`_ on GitHub to know how to use Pum. But below is a Pum quickstart,
+applied to Procrastinate.
+
+First of all, note that Pum connects to the PostgreSQL database using a `PostgreSQL
+Connection Service`_. So you have to have a Connection Service File. Here's an example
+of a Connection Service File (located at ``~/.pg_service.conf``):
+
+.. code-block::
+
+    [procrastinate]
+    host=localhost
+    port=5432
+    user=postgres
+    password=xxxxxxx
+    dbname=procrastinate
+
+You can use ``psql service=procrastinate`` to check that you can connect to
+PostgreSQL through the ``procrastinate`` service defined in your Connection Service
+File.
+
+The first time you install Procrastinate you will need to apply the Procrastinate schema
+to the database using the ``procrastinate schema --apply`` command.
+
+If you decide to use Pum for handling migrations you'll also need to create the Pum
+table (which Pum uses internally for tracking schema versions), and set the ``baseline``
+in this table.
+
+Let's say you are installing Procrastinate 0.5.0. Your baseline will then be 0.5.0. This
+the Pum command to set the baseline:
+
+.. code-block:: console
+
+    pum baseline --pg_service procrastinate --table public.pum --dir procrastinate/sql/migrations/ --baseline 0.5.0
+
+* ``--pg_service`` specifies the PostgreSQL Connection Service to use
+* ``--table`` specifies the name of the Pum table to create and insert the baseline into
+  (the table name must be prefixed by the schema name)
+* ``--dir`` specifies the path of the directory including migration scripts (it is
+  not relevant here, yet mandatory)
+* ``--baseline`` specifies the version number to use for the baseline
+
+.. note::
+
+    With Procrastinate 0.6.0 and higher you can use ``procrastinate schema
+    --migrations-path`` to get the path to the directory containing the Procrastinate
+    migration scripts.
```

Comment:
Maybe word the doc the other way around, talking about <=0.5.0

Reply:
Do you mean you'd like to see `--dir $(procrastinate schema --migrations-path)` in the Pum commands in the how-to?

Reply:
hm, yeah, you think it's a bad idea ?

Reply:
I've done how you suggested it.

Reply:
> hm, yeah, you think it's a bad idea ?

Nope, all good to me.